Jint/Native/Array/ArrayPrototype.cs

```diff
@@ -79,11 +79,9 @@ private JsValue LastIndexOf(JsValue thisObj, JsValue[] arguments)
             for (; k >= 0; k--)
             {
                 var kString = k.ToString();
```

Comment:
Change `k` to `int` and use `TypeConverter.ToString(k);` ? Should save the allocation and the string gen for most cases.

Reply:
I think I had it there but some test gave overflow, might just have needed uint or something. Worth to check again.